pkg/packages/install.go

```diff
@@ -179,25 +179,25 @@ func ShouldInstallBinary(name string) (fileName string, download bool, err error
 	pgmPath, err := exec.LookPath(fileName)
```

Comment:
This whole check should probably be removed.
Also let's remove the named returns altogether. 
This should probably also be synced with PR #6917 from @nehagup

Reply:
Let me take a look at that PR.

Reply:
Again, I think I need to wait till that PR is merged, what do u say?

Reply:
I could do this:
```
func ShouldInstallBinary(name string) (fileName string, download bool, err error) {
	fileName = BinaryWithExtension(name)
	download = false

	binDir, err := util.JXBinLocation()
	if err != nil {
		return fileName, download, err
	}

	// lets see if its been installed
	exists, err := util.FileExists(filepath.Join(binDir, fileName))
	if err != nil {
		return fileName, download, err
	}
	if exists {
		return fileName, download, err
	}
	download = true
	return fileName, download, err
}
```
It works in my case (the ksync integration test passes), I dont see the point in installing ksync in global PATH, or even check if it exists there. JXBIN should be the single source of truth. However, this function is used at lot of places, and my refactoring might break more things, and goes beyond the scope of this PR. I could also just extract this to a separate function for only ksync, but a generic function would make more sense imo.

Reply:
The `ShouldInstallbinary` part is getting taken care of by @daveconde and his work for multi- cluster support - https://github.com/jenkins-x/jx/pull/7045